service/controller/appcatalog/v1/key/key.go

```diff
@@ -0,0 +1,37 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	versionBundleAnnotation = "giantswarm.io/version-bundle"
```

Comment:
In the light of https://github.com/giantswarm/giantswarm/issues/3703 and `epic/voo` which will make version bundles actually obsolete, do we still need this?

Reply:
We are using this variable to compare between CRs version. 
Are you suggesting to use another way?

Reply:
VOO spec is to use a label `operator.giantswarm.io/version=v1.2.3`. Here we are using an annotation. I'd like us to align here and use a label too `application.giantswarm.io/version=v1.2.3`.

We went with an annotation to keep the CRs lean but I think a label is better. Its explicit and there are some cases where we would want to use it as a selector.

@puja108 @tomahawk28 WDYT?

Good to discuss but for the context of this PR lets please stick with the annotation.

Reply:
There was an update to the VOO spec. We will use annotations like `app-operator. giantswarm.io/version=v1.2.3`. The same ones we have for `nodeconfigs`. There are two benefits: (1) we already have that defined and (2) multiple operators can reconcile the same CR.

Reply:
I think `application.giantswarm.io/version=v1.2.3` makes sense. Let's just put that in `fmt`.